### PR TITLE
docs: update docs for focus change event (#784)

### PR DESCRIPTION
**Pull request by illiteratewriter**

### CODE REVIEW

**Comment by illiteratewriter** [comment]
I was also thinking of adding the following lines to the example in the docs:
```
execute!(
        stdout,
        EnableBracketedPaste,
        EnableFocusChange,
        EnableMouseCapture,
    )?;
```
so that there is better clarity in the example. Let me know if this sounds like a good addition.

File: src/event.rs [hunk 1]
```diff
@@ -17,11 +17,11 @@
...
 //! **Make sure to enable [raw mode](../terminal/index.html#raw-mode) in order for keyboard events to work properly**
 //!
-//! ## Mouse Events
+//! ## Mouse and Focus Events
 //!
-//! Mouse events are not enabled by default. You have to enable them with the
-//! [`EnableMouseCapture`](struct.EnableMouseCapture.html) command. See [Command API](../index.html#command-api)
-//! for more information.
+//! Mouse and focus events are not enabled by default. You have to enable them with the
+//! [`EnableMouseCapture`](struct.EnableMouseCapture.html)/[`EnableFocusChange`](struct.EnableFocusChange.html) command.
```
Comment (joshka) [inline src/event.rs:23]: Small nit: this could do with a space in between the links.

**Review by joshka — APPROVED** [review]
This is a simple doc change which is correct and should be merged.

**Comment by joshka** [comment]
> I was also thinking of adding the following lines to the example in the docs:

Yes - this would be a good idea. Don't forget the corresponding disable though.

**Review by joshka — APPROVED** [review]

**Review by joshka — APPROVED** [review]

**Review by TimonPost — APPROVED** [review]
Cool

**Pull request merged**